### ISSUE TITLE
Installation of picongpu

### ISSUE THREAD

**Issue by deepraj1234** [ComputationalRadiationPhysics/picongpu#2960]
i get the following error (attached in the text file) while trying the command: "$HOME/src/spack/bin/spack bootstrap"  running the command multiple times doesn't help.

[Untitled 1.txt](https://github.com/ComputationalRadiationPhysics/picongpu/files/3172785/Untitled.1.txt)

**Comment by ax3l** [comment]
Thanks for your report!
Can you open the URL http://prdownloads.sourceforge.net/tcl/tcl8.6.8-src.tar.gz directly? Looks like an issue with your internet connection when downloading `tcl`.

**Comment by deepraj1234** [comment]
when i tried to open the URL  http://prdownloads.sourceforge.net/tcl/tcl8.6.8-src.tar.gz  it downloaded the file. in the error message it is showing: "curl: (35) error:1408F10B:SSL routines:ssl3_get_record:wrong version number". is it something to do with wrong version of tcl?

**Comment by ax3l** [comment]
Are you behind a proxy server or have some kind of antivirus solution installed that intercepts your connections? An SSL error means something messes with your encrypted connections, e.g. preventing the upgrade to the URL:
https://prdownloads.sourceforge.net/tcl/tcl8.6.8-src.tar.gz (does that one work as well for you?)

**Comment by deepraj1234** [comment]
yes I am behind an authenticated proxy server. however I have exported proxy settings (with user ID and password) in the Linux terminal. in Linux terminal, "git ..."commands are fetching spack from its repository. In fact, "spack bootstrap" fetches szip and installs it, subsequently when it tries to fetch tcl, it fails.
I downloaded tcl separately and put it in repo directory where the packages are kept after fetching, still spack bootstrap does not take it from that location and tries to fetch again and fails.

**Comment by ax3l** [comment]
That's good info, thanks!
Can you report the same problem upstream with spack again, please? https://github.com/spack/spack/issues One probably has to set the proxy settings for spack somewhere.
